__device-tests__/helpers/utils.js

```diff
@@ -99,6 +99,21 @@ const setupDriver = async () => {
 		if ( isLocalEnvironment() ) {
 			desiredCaps = _.clone( iosLocal );
 			desiredCaps.app = path.resolve( localIOSAppPath );
+
+			// Set the iOS runtime to one of the available simulator runtimes
+			try {
+				const allRuntimes = childProcess.execSync( 'xcrun simctl list runtimes --json' ).toString();
+				const iOSRuntimes = JSON.parse( allRuntimes ).runtimes.filter( ( runtime ) =>
+					runtime.name.startsWith( 'iOS' )
+				);
+				const chosenRuntimeVersion = iOSRuntimes[ 0 ].version; // assume first in list should be used
```

Comment:
Comparing version strings is harder than just an alphabetic sort – and I'm not sure if there will be point (or patch versions) released (e.g. 14.5.1) – so I'm going with the first option.

I have just one runtime version installed at the moment but I'm installing another to help verify if the `xcrun simctl list runtimes --json` command used here sorts it for us.

Reply:
@SergioEstevao pinging you here in case you want to add something.

Reply:
I found out we needed to sort the `simctl` runtime list ourselves so I've updated the PR. 
This ended needing a complex solution to a small problem, so it's up to you whether we discard this – I'm not too keen on adding lots of extra complexity for what seems like little gain. Let me know what you think.